datascience/tables.py

```diff
@@ -753,9 +746,9 @@ class instance. The formatter is run each time ``__repr__`` is called. So, while
         """
         if inspect.isclass(formatter):
             formatter = formatter()
-        if callable(formatter) and not hasattr(formatter, 'format_column'):
-            formatter = _formats.FunctionFormatter(formatter)
         if not hasattr(formatter, 'format_column'):
+	    if callable(formatter):
```

Comment:
This is incorrect - the `formatter` variable was being overwritten in the current version of the code. If the overwritten still didn't have the `format_column` attr, then you'd want to raise the exception. Please revert this change - or explain further why we should introduce new logic to this function.

In either case, the repo is not compiling due to the changes here (look at the GH Action results) - so please make a change here.

Reply:
Got it, yes it is being overwritten, reverted the change